packages/manager/src/services/persistent/queries.tsx

```diff
@@ -81,3 +81,22 @@ export const useAddEnv = (propertyIdentifier?: string) => {
     }
   });
 };
+
+// POST OPERATIONS
+const deleteEphemeralEnv = async (dto: string): Promise<TDeleteEnvDTO> => {
```

Comment:
Hi @nikhitamore , please change the `dto` to `envName` in future

Reply:
done